openassessment/data.py

```diff
@@ -670,3 +672,52 @@ def collect_ora2_responses(cls, course_id, desired_statuses=None):
                 result[item_id][status] += 1
 
         return result
+
+    @classmethod
+    def collect_ora2_submission_files(cls, course_id):
+        """
+        Collects information about submission files in form dictionary
+        with following structure:
+        ```
+        {
+            <course_id>: {
+                <student_id>: {
+                    attachments: [...attachment dicts...],
+                    texts: [...text dicts...]
+                },
+                ...
+            },
+            ...
+        }
+        ```
+        """
+
+        all_submission_information = list(sub_api.get_all_course_submission_information(course_id, 'openassessment'))
```

Comment:
Why convert this to a list when it's going to be iterated over normally?

```suggestion
        all_submission_information = sub_api.get_all_course_submission_information(course_id, 'openassessment')
```

Reply:
Ah, that's a copy-paste from another function. No point in turning this into list here.